### PR TITLE
Makes IPC revival more spectacular

### PR DESCRIPTION
**Pull request by FlattestGuitar**
:cl: Flatty
add: IPCs now reboot more spectacularly.
/:cl:

beep boop

### CODE REVIEW

**Comment by AffectedArc07** [comment]
So what exactly happens?

**Comment by FlattestGuitar** [comment]
that'd be a spoiler

File: code/modules/mob/living/carbon/human/species_types/ipc.dm [hunk 1]
```diff
@@ -77,6 +78,10 @@ datum/species/ipc/on_species_loss(mob/living/carbon/C)
 		return 1
 
 /datum/species/ipc/spec_death(gibbed, mob/living/carbon/C)
+	saved_screen = C.dna.features["ipc_screen"]
+	C.dna.features["ipc_screen"] = "BSOD"
+	C.update_body()
+	sleep(3 SECONDS)
```
Comment (ike709) [inline code/modules/mob/living/carbon/human/species_types/ipc.dm]: \>using sleep()

**Comment by Fox-McCloud** [comment]
0/10, not playing [epic boot sound](https://www.youtube.com/watch?v=7nQ2oiVqKHw) when they come back online.

**Comment by SamHPurp** [comment]
I agree with @Fox-McCloud but maybe a floppy disk drive https://www.youtube.com/watch?v=7Qz9a8kYYkA

**Pull request merged**